src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.Mono.cs

```diff
@@ -202,13 +197,12 @@ public sealed class AssemblyBuilder : Assembly
         private object? permissions_minimum;
         private object? permissions_optional;
         private object? permissions_refused;
-        private PortableExecutableKinds peKind;
-        private ImageFileMachine machine;
+        private int peKind;
```

Comment:
Is this needed ?

Reply:
the field is not used and it drops additional dependencies